### PR TITLE
feat: help menu card

### LINKED ISSUE #122: Help menu
**Description**

**Acceptance Criteria**

Steps:

- step 1
- step 2

**Additional Information**
From the Figma, the result will look something like this:
image

### PR DESCRIPTION
**Pull request by vilinh**
## Developer: Vi-Linh Vu

Closes #122 

### Pull Request Summary

Created help menu component. 

### Modifications
- `src/components/help-menu.tsx`: Help menu component
- `src/styles/globals.css`: Added color values used in badges, assuming they will be used elsewhere (eg the tables).
- `src/components/admin-table/columns.tsx`: Added help menu component to appear on hover over Admin Table status badges.

### Testing Considerations

Viewed component locally. Modified it to look more aligned when inserted in HoverCardContent component because I'm assuming that's the purpose of it (having it as a child of the HoverCardContent component messes with the styling. Component itself is not aligned if just rendered on its own. (See screenshots)

### Pull Request Checklist

- [X] Code is neat, readable, and works
- [X] Comments are appropriate
- [X] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [X] The developer name is specified
- [X] The summary is completed
- [X] Assign reviewers

### Screenshots/Screencast

Screenshot 2024-04-20 at 8 05 39 PM

Component on its own (content is not lined up)

Screenshot 2024-04-20 at 8 05 23 PM

Component when hovering over badge in table

### CODE REVIEW

File: src/components/help-menu.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { Badge } from "./ui/badge";
+import { Card } from "./ui/card";
+
+export default function HelpMenu() {
+  return (
+    <Card className="flex w-[350px] gap-2 p-2">
+      <div className="flex flex-col gap-2">
+        <Badge
```
Comment (ryanhu021) [inline src/components/help-menu.tsx]: is it possible to replace these with the `StatusBadge` component so that everything is consistent?

**Review by ryanhu021 — CHANGES_REQUESTED** [review]
one comment, everything else looks good!

File: src/components/help-menu.tsx (hunk 1 above)
Comment (vilinh) [reply src/components/help-menu.tsx]: Yes. Updated:
image

**Review by ryanhu021 — APPROVED** [review]
looks good!

**Pull request merged**